ui/app/templates/vault/cluster/init.hbs

```diff
@@ -91,9 +91,8 @@
           {{/if}}
           <DownloadButton
             class="button is-ghost"
-            @data={{this.keyData}}
```

Comment:
reordered args so that download service and button are consistent: `filename`, `content/data`, `extension`